WordPress/src/main/res/values/strings.xml

```diff
@@ -75,6 +75,11 @@
     <string name="upload">Upload</string>
     <string name="learn_more">Learn more</string>
     <string name="posting_post">Posting \"%s\"</string>
+    <string name="blog_posts">Blog Posts</string>
```

Comment:
Can be prefix button strings with `btn_` so it's more obvious they're button captions? We've been very inconsistent with that in the past, but it'd be nice to be more consistent going forward. I tried to do this with [the reader strings](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/values/strings.xml#L684).